chromebook-setup.sh

```diff
@@ -687,6 +687,12 @@ cmd_deploy_kernel_only()
     cmd_eject_storage
 }
 
+if command -v dnf > /dev/null; then
```

Comment:
Maybe checking `/etc/os-release` instead ?

Reply:
Yeah that works too